ts/Core/Axis/LogarithmicAxis.ts

```diff
@@ -204,11 +209,27 @@ class LogarithmicAxisAdditions {
     }
 
     public lin2log(num: number): number {
-        return Math.pow(10, num);
+        const isNegative = num < 0 && this.axis.options.allowNegativeLog;
+
+        let result = Math.pow(10, Math.abs(num));
+
+        if (result < 10) {
+            result = (10 * (result - 1)) / (10 - 1);
+        }
+
+        return isNegative ? -result : result;
```

Comment:
This is the reason why all demos has changed, compare values in console: 
- current: https://jsfiddle.net/BlackLabel/8yrn47mj/
- candidate: https://jsfiddle.net/BlackLabel/y56fsam1/

I don't know the reasoning behind these normalizations, but perhaps we should simply use `Math.pow(10, num);` when `allowNegativeLog` is not defined. I'm referring to both `lin2log` and `log2lin` functions here :)